contrib/babelfishpg_tsql/src/tsql_for/forxml_old.c

```diff
@@ -45,6 +52,10 @@ tsql_query_to_xml(PG_FUNCTION_ARGS)
 	StringInfo result = tsql_query_to_xml_internal(query, mode,
 											element_name, binary_base64, root_name);
 
+	ereport(WARNING,
+			(errcode(ERRCODE_WARNING_DEPRECATED_FEATURE),
+			 errmsg("This version of FOR XML has been deprecated. We recommend recreating the view for this query.")));
```

Comment:
There is no test for this. You can either
1. modify our JDBC test framework to test the warning message, or
2. use installcheck